### DISCUSSION TITLE
part-ii/2019-83/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#320] [Question comments]
Part II 2019 Linear Analysis: Paper 4, Section II, H 

https://questions.tripos.org/part-ii/2019-83/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

(c) There are no eigenvalues, $\left\{|z|=1\right\}$ is the approximate spectrum. 

</details>
